testing/endtoend/evaluators/validator.go

```diff
@@ -126,6 +126,11 @@ func validatorsParticipating(_ *types.EvaluationContext, conns ...*grpc.ClientCo
 	if e2eparams.TestParams.LighthouseBeaconNodeCount != 0 {
 		expected = float32(expectedMulticlientParticipation)
 	}
+	if participation.Epoch == params.BeaconConfig().ElectraForkEpoch {
+		// The first slot of Electra will be missed due to the switching of attestation types
```

Comment:
Is this by design ? Why aren't we fixing it

Reply:
pari mentioned prysm hasn't missed any slots during the interop/devnets perhaps this is more of an issue on the participation evaluator?